src/Microsoft.PowerShell.Commands.Management/commands/management/ParsePathCommand.cs

```diff
@@ -154,18 +135,7 @@ public SwitchParameter NoQualifier
         /// </value>
         ///
         [Parameter(ParameterSetName = parentSet, Mandatory = false, ValueFromPipelineByPropertyName = true)]
-        public SwitchParameter Parent
```

Comment:
The default value of _parent used to be true. The default for a switch is False. Please revisit.